### ISSUE TITLE
Usynlige config-settings

### ISSUE THREAD

**Issue by larsjaas** [RosenborgSupporterSoftware/RBKweb-ext#64]
Jeg tenker at den dict'en som brukernavn-tracking-modulen vil ha er helt uinteressant å få presentert i settings-GUIet (bortsett fra for min egen dl undr debugging/development), så hver setting bør ha en bool om den er visible? Eller enum for visibility (always, beta, alpha, never)?

**Comment by havremunken** [comment]
Jeg liker. Dette har svirret litt i bakhodet på meg også når jeg har tenkt gjennom, og jeg liker enum'en da den gir oss fleksibilitet.

**Comment by larsjaas** [comment]
Får ha en AppContext et sted som inneholder versjonsnummeret, release-dato, git-commit, backend-versjon osv?

**Comment by havremunken** [comment]
Ja, det er en god idé. Det kan jo ev. være litt som Logger, en klasse med en statisk Instance-variabel. 

Da bumper vi fort inn i samme jeg nevnte i en annen sammenheng - behovet for å holde info i sync. Akkurat nå er det ikke noe stort problem, det ligger i package.json og manifest-fila, så må vi kanskje ha det et sted til for å fores inn i AppContext - for den kan vel kanskje ikke lese manifest-fila ved runtime? Usikker. På ett-eller-annet vis må vi ihvertfall få matet denne infoen inn i den nye greia. dotenv kan sikkert funke, men jeg ser nå etterhvert et litt økende behov for å automatisere prosessen litt.
